scout/server/blueprints/variants/templates/variants/fusion-variants.html

```diff
@@ -0,0 +1,189 @@
+    <div class="card panel-default" id="accordion">
+      <div class="card-header">
+        <strong><a data-bs-toggle="collapse" data-parent="#accordion" href="#collapseFilters">FusionFilters</a></strong>
+      </div>
+      <!--Expand filters form if filters were used in a previous POST request-->
+      <div id="collapseFilters" class="card-body panel-collapse collapse {{ 'show' if expand_search is sameas true }}">
+        {{ fusion_filters(form, institute, case) }}
+      </div>
+      {{ filter_form_footer(form, result_size, total_variants, more_variants, page, variants|length, institute) }}
+    </div>
+
+    <div class="card mt-3">
+      <table id="variantsTable" class="table table-hover table-bordered" aria-label="Fusion variants table">
+        <thead class="table-light thead">
+          <tr>
+            <th style="width:3%"></th>
+            <th style="width:4%" title="Beta version. Currently testing">Rank (beta)</th>
+            <th style="width:6%" title="Genes">Fusion Genes</th>
+            <th style="width:4%" title="Number of callers detecting a given event">Callers</th>
+            <th style="width:4%" title="Beta version. Currently testing">Score (beta)</th>
+            <th style="width:6%" title="Number of paired-ends that support the event">Junction Reads</th>
+            <th style="width:6%" title="Number of split reads that support the event">Split Reads</th>
+            <th style="width:6%" title="Fusion fragments per million total RNA-seq fragments">FFPM</th>
+            <th style="width:8%" title="Frame status of the fusion">Frame Status</th>
+            <th style="width:8%" title="Transcript ID">Transcript ID</th>
+            <th style="width:4%" title="Exon Number">Exon Number</th>
+            <th style="width:4%" title="Breakpoints">Breakpoints</th>
+            <th style="width:4%" title="Orientation">Orientation</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for variant in variants %}
+            {{ variant_row(variant) }}
+          {% else %}
+            <tr>
+                <td colspan="11">No matching variants</td>
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+  </div>
+  {{ dismiss_variants_block(dismiss_variant_options, institute, case, show_dismiss_block) }}
+</form>
+<div class="container-fluid" style="padding-bottom: 100px">
+ {{ pagination_footer(more_variants, page) }}
+</div>
+{% endblock %}
+
+{% macro variant_row(variant) %}
+  {% if variant.dismiss_variant %}
+   <tr class="dismiss">
+  {% elif 'causatives' in case and variant._id in case.causatives %}
+   <tr class="causative">
+  {% else %}
+  <tr>
+  {% endif %}
+    <td><input type="checkbox" value="{{variant._id}}" name="dismiss" onclick="enableDismiss()"></td>
+    <td class="text-start">
+      {{ cell_rank(variant, institute, case, form, manual_rank_options) }}
+    </td>
+    <td>
+      <div class="align-items-center">
+        {% if variant.genes %}
```

Comment:
Ah I see that links to genes without variant.genes (where the HGNC ID is missing) are not present:

<img width="393" alt="image" src="https://github.com/Clinical-Genomics/scout/assets/28093618/e565635f-a4b3-4238-ae05-5bb8969f246f">

Just out of curiosity, how come these variants don't have HGNC IDs??

Reply:
so the pipeline gives gene symbols and Annick added a step to retrieve hgnc_ids but some IDs they cannot be found, for example `IGH@`